docs/source/create_channel/create_channel.md

```diff
@@ -0,0 +1,383 @@
+# Creating a channel
+
+In order to create and transfer assets on a Hyperledger Fabric network, an organization needs to join a channel. Channels are a private layer of communication between specific organizations. Each channel consists of a separate ledger that can only be read and written to by channel members, who are allowed to join their peers to the channel and receive new blocks of transactions from the ordering service. While the peers, nodes, and Certificate Authorities form the physical infrastructure of the network, channels are the process by which organizations connect with each other and interact. Channels are invisible to other members of the network, making them the most powerful tool for privacy in a blockchain consortium.
+
+You can use this tutorial to learn how to create new channels using the [configtxgen](../commands/configtxgen.html) CLI tool and then use the [peer channel](../commands/peerchannel.html) commands to join a channel with your peers. While this tutorial will leverage the Fabric test network to create the new channel, the steps in this tutorial can also be used by network operators in a production environment.
+
+In the process of creating the channel, this tutorial will take you through the following steps and concepts:
+
+- [Setting up the configtxgen tool](#setting-up-the-configtxgen-tool)
+- [Using the configtx.yaml file](#the-configtx-yaml-file)
+- [The orderer system channel](#the-orderer-system-channel)
+- [Creating an application channel](#creating-an-application-channel)
+- [Joining peers to the channel](#join-peers-to-the-channel)
+- [Setting anchor peers](#set-anchor-peers)
+
+## Setting up the configtxgen tool
+
+Channels are created by building a channel genesis block that contains the initial configuration of the new channel. While it is possible to build these blocks manually, it is easier to use the [configtxgen](../commands/configtxgen.html) tool to write genesis blocks and channel creation artifacts. The tool works by reading a `configtx.yaml` file that defines the configuration of your channel, and then writing the configuration into an artifact that can be used to create the channel. Before we discuss the `configtx.yaml` file in the next section, we can get started by downloading and setting up the `configtxgen` tool.
+
+You can download the `configtxgen` binaries by following the steps to [install the samples, binaries and Docker images](../install.html). `configtxgen`, along with other Fabric tools, will be downloaded to the `bin` folder of your local clone of the `fabric-samples` repository.
```

Comment:
[that] will be downloaded

Reply:
Sentence may read better as: "`configtxgen` will be downloaded to the `bin` folder of your local clone of the `fabric-samples` repository along with other Fabric tools."